### ISSUE TITLE
How many syllables is "diktsksk" (from kiddy as in skript-kiddies)?

### ISSUE THREAD

**Issue by Windowsfreak** [paralogical/glish#5]
I'm already struggling with pronouncing Russian `Borshch` or the English `Scrunched`, but Diktsksk (from kiddy) is a new challenge!

**Comment by paralogical** [comment]
yeah, looks like there are some places where it can get stuck in loops  in the graph, leading to sksks and ststs and such. I guess it's just more personality at this point.
